source/database-mysql.rst

```diff
@@ -54,9 +54,9 @@ Tools like Adminer or phpMyAdmin provide you with a web-based way of doing that
 
 If you don't see any output, it's a good thing; MariaDB only complains if something went wrong.
 
-.. note ::
-  Please put the new password into your ``~/.my.cnf`` file with a text editor of your choice.
-  That way, MariaDB command-line tools are still able to automatically log you in.
+.. warning ::
+  It is very important to put the new password into your ``~/.my.cnf`` file with a text editor of your choice.
+  That way, MariaDB command-line tools are still able to automatically log you in. But make sure that you *really changed* the password before with the commands shown above.
```

Comment:
_It's very important to_ is enough imo. We don't need the same warning again here... People do what they want anyway 😅

Reply:
to be honest I wasn't very happy with this too. Because there are two different problems, the first sentence is for:
> user changed password with mysql but didnt update it in the `.my.cnf`
the last sentence is for
> user just changed the `.my.cnf` but did not really update it
because it is explained in detail above how to change the password in the correct way, the last problem shouldn't turn up, but it unfortunately does sometimes. However, I follow your opinion and would delete the sentence for now